### PR TITLE
doc/ceph-fuse: describe -n option

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
Fixes: https://tracker.ceph.com/issues/42300

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

File: doc/man/8/ceph-fuse.rst [hunk 1]
```diff
@@ -50,6 +50,12 @@ Any options not recognized by ceph-fuse will be passed on to libfuse.
 
    Connect to specified monitor (instead of looking through ceph.conf).
 
+.. option:: -n client.{cephx-username}
+
+   Pass the name of CephX user who's secret key has be to used for mounting.
+   This option is mandatory to when admin keyring is absent on the host where
```
Comment (jtlayton) [inline doc/man/8/ceph-fuse.rst]: The last sentence doesn't seem quite right. You need this when you want your sessions to connect as a different cephx user than "admin" (which is the default).

Comment (rishabh-d-dave) [reply doc/man/8/ceph-fuse.rst]: You are right. Fixed it.

File: doc/man/8/ceph-fuse.rst [hunk 2]
```diff
@@ -50,6 +50,10 @@ Any options not recognized by ceph-fuse will be passed on to libfuse.
 
    Connect to specified monitor (instead of looking through ceph.conf).
 
+.. option:: -n client.{cephx-username}
+
+   Pass the name of CephX user who's secret key has be to used for mounting.
```
Comment (fullerdj) [inline doc/man/8/ceph-fuse.rst]: nit: s/who's/whose/; s/has/is/

File: doc/man/8/ceph-fuse.rst [hunk 3]
```diff
@@ -16,10 +16,10 @@ Description
 ===========
 
 **ceph-fuse** is a FUSE ("Filesystem in USErspace") client for Ceph
-distributed file system. It will mount a ceph file system specified
-via the -m option or described by ceph.conf (see below) at the
-specific mount point. See `Mount CephFS using FUSE`_ for detailed
-information.
+distributed file system. It will mount a ceph file system specified via the -m
+option or described by ceph.conf (see below) at the specific mount point. By
+default, CephX user admin is used to mount CephFS. the See `Mount CephFS using
```
Comment (fullerdj) [inline doc/man/8/ceph-fuse.rst]: nit: s/the//

File: doc/man/8/ceph-fuse.rst (hunk 2 above)
Comment (rishabh-d-dave) [reply doc/man/8/ceph-fuse.rst]: Yuck! Sorry about that.

File: doc/man/8/ceph-fuse.rst (hunk 3 above)
Comment (rishabh-d-dave) [reply doc/man/8/ceph-fuse.rst]: Fixed.

**Comment by rishabh-d-dave** [comment]
jenkins render docs

File: doc/man/8/ceph-fuse.rst [hunk 4]
```diff
@@ -16,10 +16,10 @@ Description
 ===========
 
 **ceph-fuse** is a FUSE ("Filesystem in USErspace") client for Ceph
-distributed file system. It will mount a ceph file system specified
-via the -m option or described by ceph.conf (see below) at the
-specific mount point. See `Mount CephFS using FUSE`_ for detailed
-information.
+distributed file system. It will mount a ceph file system specified via the -m
+option or described by ceph.conf (see below) at the specific mount point. By
+default, CephX user admin is used to mount CephFS. See `Mount CephFS using
```
Comment (batrick) [inline doc/man/8/ceph-fuse.rst]: As I noted in the your other doc PR, let's not mention the admin key in docs.

Comment (rishabh-d-dave) [reply doc/man/8/ceph-fuse.rst]: Removed the "admin" bit here.

Comment (rishabh-d-dave) [reply doc/man/8/ceph-fuse.rst]: Although, I must ask why? I think at least man page should mention it.

**Comment by rishabh-d-dave** [comment]
jenkins render docs

File: doc/man/8/ceph-fuse.rst (hunk 4 above)
Comment (jtlayton) [reply doc/man/8/ceph-fuse.rst]: Agreed. If you don't specify a -n option then it defaults to "admin". The user ought to be informed of that default.

Comment (batrick) [reply doc/man/8/ceph-fuse.rst]: It's arguably not a good default which is why I don't want to draw attention to it in the docs. Abuse of the admin key creates problems. Mentioning it as a default encourages users to deploy the admin key everywhere.

**Pull request merged**